### PR TITLE
API endpoint

### PR DESCRIPTION
**Pull request by SergSkol**
- Create an API endpoint to list all posts for a user.
- Create an API endpoint to list all comments for a user's post.
- Create an API endpoint to add a comment to a post

### CODE REVIEW

File: app/controllers/api/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+module Api
+  class Api::CommentsController < ApiController
+    # load_and_authorize_resource
```
Comment (rustam-tolipov) [inline app/controllers/api/comments_controller.rb]: **You both did great job!**
I like the API implementations 👍 

- [ ] However, this line 3 looks odd. If this line of code is not necessary kindly remove it.

File: app/controllers/api/posts_controller.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+module Api
+  class Api::PostsController < ApiController
+    # load_and_authorize_resource
```
Comment (rustam-tolipov) [inline app/controllers/api/posts_controller.rb]: - [ ] Please remove commented code if not necessary!

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello Tea,,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_
_I like the routing of your API_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/api/comments_controller.rb (hunk 1 above)
Comment (SergSkol) [reply app/controllers/api/comments_controller.rb]: thanks! It's done

File: app/controllers/api/posts_controller.rb (hunk 2 above)
Comment (SergSkol) [reply app/controllers/api/posts_controller.rb]: thanks! It's done

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct Git Flow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**